README.md

```diff
@@ -0,0 +1,40 @@
+# testcafe-browser-provider-saucelabs
+[![Build Status](https://travis-ci.org/DevExpress/testcafe-browser-provider-saucelabs.svg)](https://travis-ci.org/DevExpress/testcafe-browser-provider-saucelabs)
+
+This is the [TestCafe](http://devexpress.github.io/testcafe) browser provider plugin for the integration with the [SauceLabs Testing Cloud](https://saucelabs.com/).
+
+## Install
+
+```
+npm install testcafe-browser-provider-saucelabs
+```
+
+## Usage
+
+Before using the provider store SauceLabs username and access key in environment variables `SAUCELABS_USERNAME` 
```

Comment:
Before using the provider, save the SauceLabs username and access key to environment variables `SAUCELABS_USERNAME` and `SAUCELABS_ACCESS_KEY`, as described in [SauceLabs documentation](https://wiki.saucelabs.com/display/DOCS/Best+Practice%3A+Use+Environment+Variables+for+Authentication+Credentials).